dev-python/semantic_version/semantic_version-2.8.5-r1.ebuild

```diff
@@ -3,7 +3,7 @@
 
```

Comment:
Good news! I've figured out that my failures were due to pytest-django. Will commit a fix.

Reply:
Great